File: c/functions/99-prime.c
#include <stdio.h>

int
prime(int num)
{
	if (num <= 1) return 0;
	for (int i = 2; i <= num/2; i++) {
		if (num % i == 0) return 0;
	}
	return 1;
}

int
main()
{
	printf("Prime numbers between 1 and 100 are:\n");

	for (int num = 1; num <=100; num++) {
		if (prime(num)) {
			printf("%d ", num);
		}
	}
	printf("\n");
	return 0;
}

File: python/prac/99-prime.py
#!/bin/python3

def prime(num):
    if num <= 1:
        return False
    for i in range(2, num // 2 + 1):
        if num % i == 0:
            return False
    return True

def main():
    print("Prime numbers between 1 and 100 are:")
    for num in range(1, 101):
        if prime(num):
            print(num, end=' ')
    print()

if __name__ == "__main__":
    main()
